ow_power_system/include/PowerSystemPack.h

```diff
@@ -29,6 +29,22 @@ using PrognoserVector = std::vector<PrognoserMap>;
 //       so it may not be addressable here.
 const int NUM_NODES = 8;
 
+// Change this value to modify the number of samples each node creates during the
+// Monte Carlo prediction process. Lower values mean faster performance, but lower
+// accuracy (needs testing for confirmation).
+// The default value is 100, but this is very slow and infeasible for the simulation.
+// My computer reached similar speeds to the original simple prognoser at a sample
+// value of 25, but this may vary from computer to computer. ~Liam
+const int NUM_SAMPLES = 100;
+
+const std::string FAULT_NAME_HPD           = "high_power_draw";
+const std::string FAULT_NAME_HPD_ACTIVATE  = "activate_high_power_draw";
+const int CUSTOM_FILE_EXPECTED_COLS           = 2;
```

Comment:
I think we agreed verbally that this should go back to 4, unless @chetankul also feels that voltage and temperature should be excluded from fault injection.  (Values placed here never seemed to have any effect, and were never explained).

Reply:
That's one of the things I have to do (and I'll request a re-review once I'm done with them):
- I need to revert the custom CSV formats to include voltage and temperature again. Still not totally sure if they do anything, but it seems like they are supposed to affect the estimation step of the GSAP prediction. They just don't directly apply the values in the file to the current status of the battery.